app/models/micropost.rb

```diff
@@ -0,0 +1,15 @@
+class Micropost < ApplicationRecord
+  belongs_to :user
+  scope :feed_sort, ->{order created_at: :desc}
+  mount_uploader :picture, PictureUploader
+  validates :user_id, presence: true
```

Comment:
```suggestion
  validates :user, presence: true
``` 
2 lệnh này khác gì nhau?

Reply:
validates :user_id nó kiểm tra xem trường user_id có tồn tại không, có nil ko, có bị khoảng trắng không 
còn thằng dưới chỉ kiểm tra xem thằng user có tồn tại không thôi ạ :3 
Trong trường hợp này mình muốn kiểm tra sự tồn tại  của thằng user_id nên mình dùng cái